source/developers-guide/shopware-5-media-service/index.md

```diff
@@ -15,7 +15,7 @@ The `Shopware\Bundle\MediaBundle` defines how Shopware manages its media files.
 
 ## The problem
 
-Since the beginning of Shopware, the media files were organized in a folder called `media` with sub-folders for each media type, e.g. image, video, and so on. Inside a sub-folder, every file has just been thrown in. The problem is that, if you have a huge amount of media files, file operations get very slow, especially on Windows systems.
+Since the beginning of Shopware, the media files were organized in a directory called `media` with sub-directories for each media type, e.g. image, video, and so on. Inside a sub-directory, every file has just been thrown in. The problem is that, if you have a huge amount of media files, file operations get very slow, especially on Windows systems.
```

Comment:
Everywhere else it's subdirectory without hyphen. Maybe remove the hyphen here as well?